android/build.gradle

```
@@ -35,10 +35,21 @@ buildscript {
   }
 }
 
+// Check the Gradle version
+def gradleVersion = GradleVersion.current()
+def requiredGradleVersion = GradleVersion.version('8.0')
+
 // Creating sources with comments
-task androidSourcesJar(type: Jar) {
-  classifier = 'sources'
-  from android.sourceSets.main.java.srcDirs
+if (gradleVersion < requiredGradleVersion) {
```

How about doing something like this instead?

```suggestion
if (GradleVersion.current() < GradleVersion.version('8.0')) {
```

---

Hi there 

Yeah that's probably better, usually I define them if the inline conditions get to long but in this case is short and understandable enough 

I have commited the changes!

Thank you